File: binance4j-blvt/src/main/java/com/binance4j/blvt/param/SubscriptionParams.java
package com.binance4j.blvt.param;

import com.binance4j.blvt.client.BLVTClient;
import com.binance4j.core.annotation.Param;
import com.binance4j.core.param.Params;

/**
 * {@link BLVTClient#subscribe} params.
 * 
 * @param tokenName The token name.
 * @param cost      Cost to subscribe to.
 */
@Param
public record SubscriptionParams(String tokenName, String cost) implements Params {
}

File: binance4j-blvt/src/main/kotlin/com/binance4j/blvt/param/SubscriptionParams.kt
package com.binance4j.blvt.param

import com.binance4j.blvt.client.BLVTClient
import com.binance4j.core.annotation.Param
import com.binance4j.core.param.Params

/**
 * [BLVTClient.subscribe] params.
 *
 * @property tokenName Token name.
 * @property cost      Cost to subscribe to.
 */
@Param
data class SubscriptionParams(var tokenName: String, var cost: String) : Params